Sources/ActiveGamesFeature/ActiveGamesView.swift

```diff
@@ -344,9 +344,9 @@ private let relativeDateFormatter = RelativeDateTimeFormatter()
       Preview {
         ScrollView {
           ActiveGamesView(
-            store: .init(
-              initialState: .init(
-                savedGames: .init(
+            store: Store(
+              initialState: ActiveGamesState(
+                savedGames: SavedGamesState(
```

Comment:
Added these when getting some compiler errors, but I don't think they were actually necessary. But also leaving them cause it doesn't hurt.